recipes/genozip/meta.yaml

```
@@ -0,0 +1,72 @@
+
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: genozip
+  version: 1.0.3
+
+source:
+  url: https://github.com/divonlan/genozip/raw/master/conda/genozip-1.0.3.tar.gz
```

You should use tags do you can download a version of the code instead of putting tarballs in git:

https://help.github.com/en/github/administering-a-repository/creating-releases

You should also use a version variable so the bot and keep things in sync and you don't risk deploying a package with a wrong version number:
```suggestion
  url: https://github.com/divonlan/genozip/archive/{{ version }}.tar.gz
```

---

Hi @chrisburr ! Thanks for checking. I fixed accordingly, I think this is good to go